pkg/asset/installconfig/ovirt/validation.go

```diff
@@ -125,3 +126,22 @@ func validateVNICProfile(platform ovirt.Platform, con *ovirtsdk.Connection) erro
 	}
 	return nil
 }
+
+// ValidateProvisioning - Perform Provisioning Validations for oVirt
```

Comment:
this does not follow the guidelines for go function comments. https://blog.golang.org/godoc

Reply:
fixed